fvtr/speccpu2006/speccpu2006.exp

```diff
@@ -19,8 +19,9 @@
 
 source ./shared.exp
 
-if { [array names env -exact SPEC] == "" } {
+if { ! [regexp "cpu2006" $env(SPEC)] } {
 	printit "Skipping: Didn't find an installed SPEC CPU 2006."
+	printit "The directory needs to have \"cpu2006\" in its name ."
```

Comment:
This change is OK, but I believe it would be better to change the commit message so that it reflects the fact that the patch affects SPEC CPU2006 as well.  Currently, the commit message gives the impression that only SPEC CPU2017 is affected.

Updating the commit message will help people understand why their environment is no longer running SPEC (if said people did not use cpu2006 in the directory name).